rebasehelper/logger.py

```diff
@@ -132,65 +182,35 @@ def add_file_handler(logger_object, path, formatter_object=None, level=None):
+    """Creates rebase-helper file handlers.
 
-    terminal_background: str = 'dark'
+    Args:
+        results_dir: Path to rebase-helper-results directory.
 
-    def set_terminal_background(self, background):
-        if background == 'auto':
-            self.terminal_background = ConsoleHelper.detect_background()
-        else:
-            self.terminal_background = background
+    Returns:
+        Path to debug log file.
 
-    def emit(self, record):
-        try:
-            message = self.format(record)
-            level_settings = self.colors[self.terminal_background].get(record.levelno, {})
-            ConsoleHelper.cprint(message, **level_settings)
-            self.flush()
-        except Exception:  # pylint: disable=broad-except
-            self.handleError(record)
-
-
-logging.setLoggerClass(CustomLogger)
-#  the main rebase-helper logger
-logger: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper')
-#  logger for output tool
-logger_output: CustomLogger = LoggerHelper.get_basic_logger('output-tool', logging.INFO)
-logger_report: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper-report', logging.INFO)
-logger_traceback: CustomLogger = LoggerHelper.get_basic_logger('traceback', CustomLogger.TRACE)
-logger_upstream: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper-upstream')
-
-console_formatter: logging.Formatter = logging.Formatter("%(levelname)s: %(message)s")
-log_formatter: logging.Formatter = logging.Formatter("%(message)s")
-debug_log_formatter: logging.Formatter = logging.Formatter(
-    "%(asctime)s %(filename)s:%(lineno)s %(funcName)s: %(message)s")
-
-main_handler: ColorizingStreamHandler = LoggerHelper.add_stream_handler(logger, logging.INFO, console_formatter)
-output_tool_handler: ColorizingStreamHandler = LoggerHelper.add_stream_handler(logger_output)
+    """
+    logs_dir = os.path.join(results_dir, constants.LOGS_DIR)
+    logger = logging.getLogger("rebasehelper")
+
+    log_formatter = logging.Formatter("%(message)s")
+    debug_log_formatter = logging.Formatter("%(asctime)s %(filename)s:%(lineno)s %(funcName)s: %(message)s")
+
+    debug_log = os.path.join(logs_dir, constants.DEBUG_LOG)
+    LoggerHelper.add_file_handler(logger, debug_log, debug_log_formatter, logging.DEBUG)
+    verbose_log = os.path.join(logs_dir, constants.VERBOSE_LOG)
+    LoggerHelper.add_file_handler(logger, verbose_log, log_formatter, CustomLogger.VERBOSE)
+    info_log = os.path.join(logs_dir, constants.INFO_LOG)
+    LoggerHelper.add_file_handler(logger, info_log, log_formatter, logging.INFO)
+    return debug_log
+
+
+def create_stream_handlers() -> Tuple[ColorizingStreamHandler, ColorizingStreamHandler]:
+    logger = logging.getLogger('rebasehelper')
+    formatter = logging.Formatter("%(levelname)s: %(message)s")
+    main = LoggerHelper.add_stream_handler(logger, logging.INFO, formatter)
+    logger_output = LoggerHelper.get_logger('special.output_tool', logging.INFO)
```

Comment:
The name should definitely start with `rebasehelper` to prevent potential conflicts with loggers from other (3rd party) modules. Actually, I don't think `logger_output` should be defined here. It should be private attribute/implementation detail of `text` output tool.

Reply:
`report` and `output_tool` logger seem to be different things. `output_tool` is used to display information to the console at then end of then run, `report` is used to create `report.txt`. I think `rebasehelper._output_tool_` looks a bit weird so I would personally just use `rebasehelper.report` and `rebasehelper.output_tool`. Do you agree?

Reply:
True, but `output_tool` is really unfortunate name. What about `rebasehelper.summary` instead?